packages/infrastructure/eslint-config/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@k8slens/eslint-config",
-  "version": "^6.5.0-alpha.0",
+  "version": null,
```

Comment:
this is odd

Reply:
Yes, I think the previous version is not valid